widgets/executionsStatus/src/types.ts

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/prefer-default-export
+
+export namespace ExecutionsStatusWidget {
+    export type Configuration = {
+        pollingTime: number;
+    };
+    export interface Params {
+        /* eslint-disable camelcase */
+        blueprint_id: Stage.ContextEntries['blueprintId'];
+        deployment_id: Stage.ContextEntries['deploymentId'] | undefined;
+        id: Stage.ContextEntries['executionId'];
+        status_display: Stage.ContextEntries['executionStatus'];
+    }
+
+    interface DataItem {
+        executions: number;
+        status_display: string;
+    }
+
+    export interface Data {
+        items: DataItem[];
+    }
```

Comment:
I think you can use `Stage.Types.PaginatedResponse`:
```suggestion
    interface DataItem {
        executions: number;
        status_display: string;
    }

    export type Data = Stage.Types.PaginatedResponse<DataItem>;
```
or even just:
```suggestion
    export type Data = Stage.Types.PaginatedResponse<{ executions: number; status_display: string }>;
```

Reply:
Whoa!
Now I don't know why should I do that! 😁 

Thanks for sending that type, looks very handy!
I'll implement that right away!

Reply:
Done 😎